client-v2/src/components/FrontsEdit/Edit.tsx

```diff
@@ -14,7 +14,7 @@ import {
 } from 'bundles/frontsUIBundle';
 import { State } from 'types/State';
 import { ActionError } from 'types/Action';
-import FrontContainer from './FrontContainer';
+import FrontContainer from './FrontSection';
```

Comment:
I've split up the files responsible for rendering a front into

`FrontSection` --> `FrontContainer` --> `FrontContent` & `Overview`
(preview bar)     (expand btns, overview)   (collections etc.)

Hopefully this is clear, super happy to hear anyone's thoughts 👍